frontend/src/pages/GuestAccessPage.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
I think I'd like to see tests covering formik's validation logic, but let's leave that for when this form is finalized.